app/core/templatetags/tickets.py

```diff
@@ -0,0 +1,13 @@
+from django import template
+from django.template.defaultfilters import stringfilter
+
+import markdown as md
+
+register = template.Library()
+
+
+@register.filter()
+@stringfilter
+def markdown(value):
+
+    return md.markdown(value, extensions=['markdown.extensions.fenced_code', 'codehilite'])
```

Comment:
scratch markdown render from model. use this renderer for all ticket related markdown.

required as this will do the ticket prettying of items. i.e. links, tickets etc.

Reply:
scratched from model to use templatetags so it can be used by all ticket models